ecl2df/summary.py

```diff
@@ -235,22 +248,38 @@ def df(
             eclfiles.get_eclsum().dates, time_index, True, start_date, end_date
         )
     else:
+        # Can be None.
         time_index_arg = time_index
 
+    if isinstance(time_index_arg, list):
+        if len(time_index_arg) < 6:
+            time_index_str = str(time_index_arg)
+        else:
+            time_index_str = str(time_index_arg[0:3] + ["..."] + time_index_arg[-3:])
```

Comment:
This is for presentational purposes I assume? Note that `[...]` is special notation in python, so I think
```python
f"{time_index_arg[0:3]} … {time_index_arg[-3:]}"
```

Might be less confusing?

Reply:
Suggestion incorporated!